packages/orbit-components/src/Toast/index.jsx.flow

```diff
@@ -0,0 +1,55 @@
+// @flow
+/*
+  DOCUMENTATION: https://orbit.kiwi/components/toast/
+*/
+import * as React from "react";
+
+import type { Globals } from "../common/common.js.flow";
+
+export type Placement =
+  | "top-left"
+  | "top-center"
+  | "top-right"
+  | "bottom-left"
+  | "bottom-center"
+  | "bottom-right";
+
+export type Toast = {|
+  +icon?: React.Element<any>,
+  +visible?: boolean,
+  +children: React.Node,
+  +dismissTimeout?: number,
+  +onMouseEnter: () => void,
+  +onMouseLeave: () => void,
+  +onDismiss: () => void,
+  +placement: Placement,
+  +offset?: number,
+  +ariaLive?: string,
```

Comment:
Hm, as far as i know it's recommended to use role `status`. Like [here](https://www.scottohara.me/blog/2019/07/08/a-toast-to-a11y-toasts.html)

Reply:
My bad, I meant `"status"`, yes. My question still stands, the idea is to prevent people from not assigning `role` and `ariaLive`. So what do you think?

Reply:
My bad, `"status"`. Should `role` default to `"status"`, then? Or should `role` not be a prop at all, if there is no case where Toast would have a different role? One less thing to type 🤷